### DISCUSSION TITLE
Skins Dropping

### DISCUSSION THREAD

**Discussion by mlanci** [SFX-WoW/Masque#205] [Issues] [answered]
I'm using Dominos and the XP/rep bar and class bar stays skinned.. but the dominos action bars just resets to default within like minutes of me changing everything to whatever color.

Actually just realized.. it drops when I enable the skin back and either re-enter stealth or go unstealth.

**Comment by StormFX** [comment]
You'll have to be more clear. You're setting custom colors and the colors are reverting or...

**Comment by mlanci** [comment]
Yes I'm going under dominos settings within MSQ and setting the background and other colors to all black. When I change stances as rogue, the colors reset and go back to default. there may be other cases that trigger but this is what I noticed so far.

**Comment by mlanci** [comment]
Just to be more clear now that i'm in game.. When I go to /msq in global and in the dominos / action bar settings I have enabled backdrop and set to a color of black as well as colors/normal and set that to black.

**Comment by StormFX** [comment, marked as answer]
I am unable to reproduce this issue. I loaded my 120 Rogue with _Dominos_ + _Masque_, enabled "Show Empty Buttons" in _Dominos_, enabled the Backdrop in _Masque_ and set the color for the Backdrop and Normal textures to black. It doesn't change or revert when going into stealth or leaving stealth.

All I can suggest at this point is that you disable all add-ons except _Masque_ and _Dominos_ and then reinstall them if necessary. You may have to reset their profiles (rename the WTF folder temporarily).

**Comment by StormFX** [comment]
Any update on this?
